hazel/hazel.bzl

```diff
@@ -131,6 +140,10 @@ def hazel_repositories(
           specified).
         - output, type, stripPrefix: Only when "url" is specified. See the
           download_and_extract documentation.
+        - patches: list of labels, patch files to apply after unpacking.
+        - patch_tool: string, patch executable.
+        - patch_args: list of strings, extra arguments to patch tool.
+        - patch_cmds: list of strings, commands to apply after patches.
```

Comment:
What’s the semantics of this? Sounds a bit vague.

Reply:
These are forwarded to the [`patch` function](https://github.com/bazelbuild/bazel/blob/b941e0b939c18257107f7f4b5186ae3a4b770eac/tools/build_defs/repo/utils.bzl#L97) from `bazel_tools`. The commands are executed after the patch was applied. I've extended the docstrings based on the [`bazel_tools` ones]( https://github.com/bazelbuild/bazel/blob/b941e0b939c18257107f7f4b5186ae3a4b770eac/tools/build_defs/repo/http.bzl#L196). Hopefully, that makes it clearer.

Reply:
Hm, not really. I think since it’s a leaky abstraction, you should just add “These are forwarded to the patch function from bazel_tools. \<link\>”  verbatim.

Reply:
@Profpatsch Added such a sentence and link. PTAL

Reply:
Looks good